kedro-training/docs/01_prerequisites.md

```diff
@@ -1,3 +1,51 @@
+# Training pre-requisites
+This Kedro training materials assume the following pre-requisites for your technical understanding.
+
+We assume understanding of these technologies and concepts during the training. In order to maximise your learning experience from the Kedro training, please review them before going through the training (there are external resource links for you to get a refresher on each topic).
+
+- Intermediate Python knowledge
```

Comment:
You say 'intermediate' here and then follow with 'basic python' which is a little confusing.

How about changing the link that says "Basic Python" to say "Python fundamentals" ?